demos/SDO-OH-EXF-dev-simple/README.md

```diff
@@ -0,0 +1,97 @@
+# Demonstration of SDO, OH, and EXF
+
+## Introduction
+
+This is a simple demonstration geared towards developers.  It begins with the Open Horizon all-in-one installation (including SDO services) in a single VM.  Running the modified SDO script, it will boot a simulated device and then install and configure an Open Horizon Agent.  The Agent will then contact the Management Hub and receive a proposed agreement to install and run an EdgeX Foundry (EXF) instance.  EXF, in turn, will be configured to run a Random Number Device service.
+
+## Requirements
+
+* An x86_64/amd64-based VM with Ubuntu 18.04 Server, 1 vCPU, 4GB RAM, and 20GB disk
+
+## Instructions
+
+### Step One: Prepare the OS
+
+* Boot the VM and log in.  Become root with `sudo -i`.
+* **Optional** - Update the OS and install any pre-requisites
+``` shell
+apt-get -y update
+apt-get -y upgrade
+apt-get -y install jq
+```
+
+### Step Two: Install Open Horizon
+
+> If Open Horizon is already installed and configured properly, skip step two.
+
```

Comment:
This step will install docker and docker-compose if those are not present on the system.

Reply:
Thanks.  I've added that note.